src/auth/registration/MarketingOptIn.tsx

```diff
@@ -1,21 +1,17 @@
 import { Link, Stack, Typography } from "@mui/material";
-import { Link as RouterLink } from "react-router-dom";
 import { FormikCheckbox } from "form/FormikHelpers";
 import { useLingui } from "@lingui/react";
 import { msg, Trans } from "@lingui/macro";
 import { PropsWithChildren } from "react";
 
 export function MarketingOptIn() {
   const { _ } = useLingui();
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const PolicyLink = (props: { to: string } & PropsWithChildren) => (
     <Link
```

Comment:
Can we not make this an proper `<a />` element rather than rely on javascript?  Seems silly to use javascript for something as basic as opening a link.

e.g. 

```
<Link component="a" href={props.to} target="_blank" rel="noopener">
...
```